### PR TITLE
Pedram/feature/add booking/form

### PR DESCRIPTION
**Pull request by pedram-am**
This introduces a new feature to the Bookings component, enabling users to add bookings via a form. The form includes input fields for first name, last name, email, title, room ID, check-in date, and check-out date. Also, resolves an issue where the last name (surname) field was not displayed in the booking table, by updating "surname" to "lastName" in the Booking.jsx file.

### CODE REVIEW

**Review by HadikaMalik — CHANGES_REQUESTED** [review]
Can we leave the surname as surname and not change it to lastName? I am asking becasue the data we are fetching, that has surname, and everything is implemented with surname. So if we change this here, then we will have to change it everywhere else as well. Let me know, until then I won't merge it. :)

File: src/components/Booking/Booking.jsx
Comment (PERicci) [inline src/components/Booking/Booking.jsx]: I suggest keeping the surname the same. It matches how we've done things in other parts of the app and stays true to the original data. This way, everything stays consistent and follows the same pattern.

**Review by PERicci — CHANGES_REQUESTED** [review]
The implementation is working well. The only tweak I'd suggest is reverting the property name from 'lastName' back to 'surname'. It's consistent with other components and aligns with the original data. Also, 'surname' feels more fitting, especially for people with multiple surnames.

**Comment by PERicci** [comment]
We may need to make some adjustments due to the implementation of the search functionality. I think it would be easier to address these after merging.

**Comment by pedram-am** [comment]
Hi, thank you for your feedback! We've decided to stick with 'surname' as it helps to keep things consistent and makes the work easier. I've gone ahead and updated the Bookings.jsx file accordingly. Thanks again! 👍

**Review by PERicci — APPROVED** [review]
Merged the changes after resolving conflicts.

**Review by PERicci — APPROVED** [review]

**Pull request merged**
